digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1558,12 +1558,13 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
       ).getOrElse(roadAddress.startAddrMValue, roadAddress.endAddrMValue, roadAddress.startMValue, roadAddress.endMValue)
     }
 
-    split.flatMap(pl =>
+    split.flatMap(pl => {
+      val floatingValue = roadAddress.validTo.isDefined && roadAddress.validTo.get.isBeforeNow
       pl.status match {
```

Comment:
I think it should be enough to check that valid_to is just set. We should never set valid_to date to the future so this isBeforeNow is not needed.